GraphsV2/utils/helper.js

```diff
@@ -224,15 +224,16 @@ export const scaleColor = (props, data, defaultColumn, setGraphColor) => {
     return colorList;
 }
 
+
 export const renderMessage = (props) => {
     const {
-        data,
+        classes,
         message,
         id
     } = props;
-    const messageClass = data && data.classes && data.classes.messageClass;
+    const messageClass = classes && classes.messageClass;
 
-    return (
+     return (
         <div id={`${id}-message`} className={messageClass ? messageClass : "center-text"}>
```

Comment:
I would like to be able to add to the classes however keep the 'center-text'. Let's change to `center-text (messageClass || '')`